omod/src/main/resources/config.xml

```diff
@@ -29,6 +29,9 @@
 	<!-- /Extensions -->
 
 	<require_version>1.7</require_version>
+    <require_modules>
```

Comment:
Should we have both require and aware of the same module?

Reply:
You got me confused earlier, this is why initially I had not added this, I think it should require and not be aware of

Reply:
Updated